src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -1488,19 +1482,19 @@ public PropertyDescriptor getColumn(Domain domain, String columnName)
         List<Map<String, Object>> validators = (ArrayList<Map<String, Object>>)column.getAllProperties().get("propertyValidators");
         Map<String, Object> validator = validators.stream()
                 .filter(a-> a.get("name").equals(name))
-                .collect(Collectors.toList())
-                .get(0);
+                .findFirst().orElse(null);
+        assertNotNull("did not find expected validator ["+name+"] on column", validator);
         return validator;
     }
 
     public Map<String, Object> getConditionalFormats(PropertyDescriptor column, String filterExpression)
     {
-        List<Map<String, Object>> validators = (ArrayList<Map<String, Object>>)column.getAllProperties().get("conditionalFormats");
-        Map<String, Object> validator = validators.stream()
+        List<Map<String, Object>> formats = (ArrayList<Map<String, Object>>)column.getAllProperties().get("conditionalFormats");
+        Map<String, Object> conditionalFormat = formats.stream()
                 .filter(a-> a.get("filter").equals(filterExpression))
-                .collect(Collectors.toList())
-                .get(0);
-        return validator;
+                .findFirst().orElse(null);
+        assertNotNull("did not find expected conditionalFormat ["+filterExpression+"] on column", conditionalFormat);
```

Comment:
Include properties in failure message.
```suggestion
        assertNotNull("did not find conditionalFormat ["+name+"] on column. Column properties: " + column.getAllProperties().toString(), validator);
```